.github/workflows/publish-provisioning-test-results.yaml

```diff
@@ -0,0 +1,48 @@
+name: Publish Provisioning Tests Results
+on:
+  workflow_run:
+    workflows: ["Build Pull Request"]
+    types:
+      - completed
+permissions: {}
+
+jobs:
+  publish-provisioning-test-results:
+    runs-on: runs-on,runner=4cpu-linux-x64,image=ubuntu22-full-x64,run-id=${{ github.run_id }}
```

Comment:
If possible, using containerized runners is cheaper. You have to set a container and install dependencies if they are not already there.
```
runs-on: org-${{ github.repository_owner_id }}-{{ arch }}-k8s
```